docs/release-procedure.md

```diff
@@ -1,3 +1,3 @@
 Distributed follows a similar procedure for releasing as the core Dask project.
 
-See https://github.com/dask/dask/blob/master/docs/release-procedure.md for instructions.
\ No newline at end of file
+See https://github.com/dask/dask/blob/master/docs/release-procedure.md for instructions.
```

Comment:
```suggestion
See https://github.com/dask/dask/blob/main/docs/release-procedure.md for instructions.
```

Reply:
I am purposefully only doing distributed in this PR. I wanted to be able to merge this one first, wait a bit and then do dask/dask.

Reply:
Gotcha, thanks for clarifying